myosuite/envs/myo/walk_v0.py

```diff
@@ -55,26 +55,9 @@ def _setup(self,
                 **kwargs,
                 )
         self.init_qpos[:] = self.sim.model.key_qpos[0]
```

Comment:
are the key-qpos/qvel assumed to be always present or those could be optional?

Reply:
they are assumed to be present. there is a reset = [None, init, random] option that either uses:
1. None ->  key_qpos[0], which is just standing
2. init -> key_qpos[2], which is flexed left knee
3. random -> key_qpos[2] and key_qpos[3]. This switches randomly between knees and also injects some more noise into the initial qpos